src/components/Menu/styles.tsx

```diff
@@ -37,6 +37,7 @@ export const MenuStyle = styled.div`
 export const MenuBodyStyle = styled.div`
   display: block;
   margin: 0 2.5rem;
+  padding-bottom: 200px;
```

Comment:
It looks like this is doing what line 32 is trying to do, can we also remove the `margin-bottom: 6rem` from line 32 of this file.

Reply:
Thanks so much Jacob!! I ended up changing it to `padding-bottom: 9rem` to give it extra space for viewing the page last updated text. Staging site should up in a bit